css/style.css

```diff
@@ -1,19 +1,168 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+:root {
+  --grey-dark: #292b2c;
+  --grey-mid: #808080;
+  --grey-light: #e4e4e4;
+  --orange-dark: #c05326;
+  --orange-light: #f7eae4;
+  --white: #fff;
+}
+
+body {
+  margin: 0;
+}
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+h1,
+p {
+  font-weight: 200;
+}
+
+h3,
+h4 {
+  font-weight: 400;
+}
+
+.header,
+.content_wrapper {
+  max-width: 1200px;
+  min-height: 60px;
+  margin: 0 auto;
+  padding: 1rem;
+  text-align: center;
+}
+
+/*  ____________  header______________________ */
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  max-height: 60px;
+}
+
+ul {
+  padding: 5px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav_item {
+  list-style: none;
+  padding: 5px 10px;
+}
+
+.nav_list {
+  color: var(--grey-mid);
+  text-decoration: none;
+}
+
+.nav_list:active {
+  color: var(--grey-dark);
+}
+
+.nav_list:hover {
+  color: var(--orange-dark);
+}
+/*  ____________  big poster section  ___________________ */
+
+.hero {
+  background-image: url("../img/first-background.jpg");
+  background-color: var(--grey-light);
+  margin-bottom: 50px;
+  color: white;
+  text-align: center;
+  padding: 30px 10px;
+  background-size: cover;
+  background-position-x: center;
+  background-repeat: no-repeat;
+  height: 75vh;
```

Comment:
I think this is a bit big compared to the example image, you don't need to perfectly match the designs but it is it good practice because sometimes it is important as a web developer (although usually you won't just have an image that you have to try to match!)